### PR TITLE
Players no longer vote "Continue playing" by default

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Players who did not vote in a restart vote no longer count towards those who voted "Continue playing" automatically.

## Why It's Good For The Game

As it is at the moment, the restart vote system is pointless, and is impossible to use even in the worst of situations when there are no admins on due to the amount of AFK players typically large enough to prevent a majority. This PR solves this issue.

It would merely allow the restart vote to be successful when it is needed - as it stands, in most cases enough people would still vote continue playing to make sure the game continues, so it won't lead to an increase in restart-griefing.

## Changelog
:cl:NewSta
tweak: players no longer vote "Continue playing" by default
/:cl:

### CODE REVIEW

**Comment by Dawson1917** [comment]
time to watch the round reset and cancel out every antag's greentext every time someone murderbones in a round and people get mad

tldr bad idea don't do this

**Comment by europaisch** [comment]
Most people would just vote continue playing I believe. It is at least worth testmerging in my opinion.

**Comment by Dawson1917** [comment]
Most people in a round who die / see everyone is dead aren't bothered about whether the round ends or not, or aren't bothered, or miss the vote popup, whereas people who do want the round to end absolutely will vote for a restart. This is a problem on smaller servers that don't have admins on 24/7, and will be a problem for TG too during hours where there aren't many admins on. Restart votes happen EVERY round someone pulls off a murderbone, and getting your antag round ended early and your greentext deleted because "i ded restart" isn't fun at all

**Comment by europaisch** [comment]
As is the future is absolutely useless though, so this would make it more useful.

**Comment by XDTM** [comment]
If a restart vote passes because of a murderbone, maybe the problem is that one player is having fun at the expense of everyone else and maybe it's ok that it passes

**Comment by europaisch** [comment]
And even so, players kinda enjoy watching them

**Comment by Dawson1917** [comment]
> 
> 
> If a restart vote passes because of a murderbone, maybe the problem is that one player is having fun at the expense of everyone else and maybe it's ok that it passes
> 
> Ok, you're super robust and won the game, let the others actually play now

I don't disagree with this but this is how TG is. Unless there's enormous rule and policy changes then this PR will conflict with currently commonly expected antag behaviour. Beyond running in hallways killing everybody with a stetchkin and cheese, there'll also be restarts every time a malf AI creates a plasma flood or a singulo/tesla is released, when virology or a sentient disease kill off half the station, or any number of other antag strategies that require/create a high kill count

**Comment by HomocidlePancakes** [comment]
We already had this PR and it got denied. Nothing has changed since then to have it merged, but you are free to try.

**Comment by europaisch** [comment]
> We already had this PR and it got denied. Nothing has changed since then to have it merged, but you are free to try.

Haven't seen that one, to be honest.

But I guess itš worth a try anyway

**Comment by JJRcop** [comment]
This PR explicitly breaks one of the config options and causes unedocumented behavior when used 

https://github.com/tgstation/tgstation/pull/15941#issuecomment-192776782

https://github.com/tgstation/tgstation/blob/345e33320b4ae76843cfa4ab60d3d96341d3a538/config/config.txt#L182-L183

This entire code block you're editing does not get run if that option is enabled in the config.

https://github.com/tgstation/tgstation/blob/345e33320b4ae76843cfa4ab60d3d96341d3a538/code/controllers/subsystem/vote.dm#L56-L57

**Comment by europaisch** [comment]
Appreciate the explanation. Closing the PR then.

**Comment by optimumtact** [comment]
Specifically the headadmins, not the keyholders
